libcontainer/cgroups/fs/blkio_test.go

```diff
@@ -210,23 +211,24 @@ func TestBlkioSetWeightDevice(t *testing.T) {
 	wd := configs.NewWeightDevice(8, 0, 500, 0)
 	weightDeviceAfter := wd.WeightString()
 
+	blkio := &BlkioGroup{}
+	blkio.detectWeightFilenames(helper.CgroupPath)
 	helper.writeFileContents(map[string]string{
-		"blkio.weight_device": weightDeviceBefore,
+		blkio.weightDeviceFilename: weightDeviceBefore,
 	})
 
 	helper.CgroupData.config.Resources.BlkioWeightDevice = []*configs.WeightDevice{wd}
-	blkio := &BlkioGroup{}
 	if err := blkio.Set(helper.CgroupPath, helper.CgroupData.config.Resources); err != nil {
 		t.Fatal(err)
 	}
 
-	value, err := fscommon.GetCgroupParamString(helper.CgroupPath, "blkio.weight_device")
+	value, err := fscommon.GetCgroupParamString(helper.CgroupPath, blkio.weightDeviceFilename)
 	if err != nil {
-		t.Fatalf("Failed to parse blkio.weight_device - %s", err)
+		t.Fatalf("Failed to parse %s - %s", blkio.weightDeviceFilename, err)
```

Comment:
So, I am working on that in https://github.com/opencontainers/runc/pull/3011 but if you want to modify this, I'd rather do something like
```suggestion
		t.Fatal(err)
```
because the error message from `fscommon.Get*` functions already contains enough context (filename).

In fact you can borrow the whole or a part of https://github.com/opencontainers/runc/pull/3011/commits/ac5ca790f10974c0dcb2563fea071abdf7fce3e3, put it before your commit so it becomes shorter.

Reply:
Switched to `t.Fatal(err)`'s in both cases that were touched to align with your work. Tested them both with deliberately wrong filenames. Error messages look good, as you say.
```
--- FAIL: TestBlkioSetWeightDevice (0.00s)
blkio_test.go:227: open /tmp/cgroup_test459287435/blkio/blkio.bfq.weight_devicey: no such file or directory
```